detect_variants/detect_variants.cwl

```diff
@@ -51,13 +51,21 @@ steps:
             normal_bam: normal_bam
         out:
             [merged_vcf]
+    pindel:
+        run: ../pindel/workflow.cwl
+        in:
+            reference: reference
+            tumor_bam: tumor_bam
+            normal_bam: normal_bam
+            interval_list: interval_list
     combine:
         run: combine_variants.cwl
         in:
             reference: reference
             mutect_vcf: mutect/merged_vcf
             strelka_vcf: strelka/merged_vcf
             varscan_vcf: varscan/merged_vcf
+            pindel_vcf: pindel/merged_vcf
```

Comment:
[combine_variants.cwl](https://github.com/dufeiyu/arvados_trial/blob/e6ac2280f36f255486e6534571791a74096050e8/detect_variants/combine_variants.cwl) will need to be updated to account for this change.